src/Serilog.Sinks.Couchbase/Sinks/Couchbase/CouchbaseSink.cs

```diff
@@ -14,22 +14,30 @@
 
 using System;
 using System.Collections.Generic;
-using System.Threading.Tasks;
 using Serilog.Debugging;
 using Serilog.Sinks.PeriodicBatching;
-using Couchbase.Extensions;
-using Couchbase.Management;
 using LogEvent = Serilog.Sinks.Couchbase.Data.LogEvent;
 
 namespace Serilog.Sinks.Couchbase
 {
+    using System.Linq;
+    using global::Couchbase;
+    using global::Couchbase.Configuration;
+    using global::Couchbase.Configuration.Client;
+    using global::Couchbase.Core;
+
     /// <summary>
     /// Writes log events as documents to a Couchbase database.
     /// </summary>
     public class CouchbaseSink : PeriodicBatchingSink
     {
         readonly IFormatProvider _formatProvider;
-        readonly global::Couchbase.CouchbaseClient _couchbaseClient;
+
+        private readonly Cluster _couchbaseCluster;
```

Comment:
For consistency, the Seriliog codebase doesn't use any explicit default modifiers (`private` on fields or `internal` on classes).